flixel/FlxSprite.hx

```diff
@@ -56,6 +57,7 @@ class FlxSprite extends FlxObject
 	 */
 	public var useFramePixels(default, set):Bool = true;
 	
+	// TODO: rename this to smoothing...
```

Comment:
Hasn't it been like that since Flixel 2 (AS3) at least? Could always add a note about that to the docs.

If we do decide to change it, we should probably keep `antialiasing` as an alias for `smoothing` and mark it as `@:deprecated` to remain backwards-compatible.